src/CoreWCF.UnixDomainSocket/src/CoreWCF/Channels/UnixDomainSocketTransportBindingElement.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Authentication.ExtendedProtection;
+using CoreWCF.Configuration;
+
+namespace CoreWCF.Channels
+{
+    public class UnixDomainSocketTransportBindingElement : ConnectionOrientedTransportBindingElement
+    {
+        private int _listenBacklog;
+        private ExtendedProtectionPolicy _extendedProtectionPolicy;
+
+        public UnixDomainSocketTransportBindingElement() : base()
+        {
+            _listenBacklog = UnixDomainTransportDefaults.GetListenBacklog();
+            //Is this needed ?
+            ConnectionPoolSettings = new UnixDomainSocketConnectionPoolSettings();
+            _extendedProtectionPolicy = ChannelBindingUtility.DefaultPolicy;
```

Comment:
I don't think it's needed for UDS

Reply:
This should get configured on the WindowsStreamSecurityBindingElement to configure the NegotiateStream instance. So I was wrong, it is needed for UDS, but I would need to look into things more to know if this is the right place or if it exists on WindowsStreamSecurityBindingElement and that's where it should be set.